src/modules/user.ts

```diff
@@ -60,11 +61,6 @@ export interface OrganizationSettings {
 
 export type UpdateOrganizationSettings = Partial<OrganizationSettings>;
 
-export interface IOrganization extends Entity, OrganizationSettings {
```

Comment:
Why did you replace the IOrganization model when we already have one that works?

Reply:
I was trying to be smart and move organization stuff into the organizations file, will move back